functions/utils/getApolloClient.ts

```diff
@@ -0,0 +1,21 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+const GRAPHQL_ENDPOINT = 'https://api.uniswap.org/v1/graphql'
+
+export const getApolloClient = () => {
+  return new ApolloClient({
+    connectToDevTools: true,
+    uri: GRAPHQL_ENDPOINT,
+    headers: {
+      'Content-Type': 'application/json',
+      Origin: 'https://app.uniswap.org',
+      'User-Agent':
+        'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.5735.110 Safari/537.36',
+    },
+    cache: new InMemoryCache(),
+    defaultOptions: {
+      watchQuery: {
+        fetchPolicy: 'cache-and-network',
```

Comment:
Why not `cache-first`? [docs](https://www.apollographql.com/docs/react/data/queries/#supported-fetch-policies)

Reply:
`cache-first` would make sense, but also if we're using cloudflare's cache wouldn't `network-only` be better?

Reply:
this refers to the in-memory cache (as you've set it), so I think you want both.

Reply:
yeah your right. i guess we can always change it in the cache pr